.github/workflows/static.yml

```diff
@@ -16,7 +16,7 @@ jobs:
         uses: actions/checkout@v2
 
       - name: PHPStan
-        uses: OskarStark/phpstan-ga@0.12.28
+        uses: OskarStark/phpstan-ga@0.12.41
```

Comment:
I dont mind this, but as you can see in https://github.com/OskarStark/phpstan-ga/issues/31, we are already running 0.12.42 =)

Reply:
0.12.42 isn't available on docker hub.

Reply:
So I couldn't use it to generate the baseline locally.